app/forms/gpo_verify_form.rb

```diff
@@ -72,7 +78,7 @@ def reset_sensitive_fields
 
   def pending_in_person_enrollment?
     return false unless IdentityConfig.store.in_person_proofing_enabled
-    user.pending_in_person_enrollment.present?
+    ProofingComponent.find_by(user: user)&.document_check == Idp::Constants::Vendors::USPS
```

Comment:
Since the proofing components should be "snapshot" onto the profile by this point, I think we may want to use that value, to avoid potential issues where a user somehow goes through identity verification again.

```suggestion
    pending_profile.proofing_components.document_check == Idp::Constants::Vendors::USPS
```

Reply:
borderline blocker, but probably fine without

Reply:
Fixed this in 64f02808f